content/en/news/releases/1.15.x/announcing-1.15.3/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Announcing Istio 1.15.3
+linktitle: 1.15.3
+subtitle: Patch Release
+description: Istio 1.15.3 patch release.
+publishdate: 2022-10-26
```

Comment:
Haven't looked, but did we release this today?

Reply:
No did not release yesterday.